.github/workflows/autopep8.yml

```
@@ -0,0 +1,23 @@
+name: autopep8
+on: pull_request
+jobs:
+  autopep8:
+    # Check if the PR is not from a fork
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - name: autopep8
+        id: autopep8
+        uses: peter-evans/autopep8@v1
+        with:
+          args: --exit-code --recursive --in-place --aggressive --aggressive .
+      - name: Commit autopep8 changes
+        if: steps.autopep8.outputs.exit-code == 2
+        run: |
+          git config --global user.name 'honomara ojisan'
+          git config --global user.email 'honomara-ojisan@users.noreply.github.com'
+          git commit -am "Automated autopep8 fixes"
+          git push
```

フォーマットがコミット履歴に残るのはあんまり良くない気がするから、formatがおかしいときはfailするだけに留めておこう

```suggestion
        run: exit 1
```